<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_tarjeta_emeplado_reporte</class>
 <widget class="QWidget" name="admin_tarjeta_emeplado_reporte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">admin_tarjeta_emeplado_reporte 
{
	background-color: rgb(244, 247, 244);
	border-color: rgb(255, 255, 0);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Recursos.qrc">:/iconos/resources/camarero.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_nombre">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>12</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>[Nombre_Mesero]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tipoEmpleado">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>51</y>
     <width>157</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #A6A9A5;
</string>
   </property>
   <property name="text">
    <string>[TIPO_EMPLEADO]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_numComandas">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>39</y>
     <width>197</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: #A6A9A5;
</string>
   </property>
   <property name="text">
    <string>[NÚMERO_COMANDAS]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_numPlatillos">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>39</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: #A6A9A5;
</string>
   </property>
   <property name="text">
    <string>[NÚMERO_PLATILLOS]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_numPersonas">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>39</y>
     <width>194</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: #A6A9A5;
</string>
   </property>
   <property name="text">
    <string>[NÚMERO_PERSONAS]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 247, 244);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tipoUsuario_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>12</y>
     <width>123</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Núm comandas</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoUsuario_5">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>12</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Núm Platillos</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoUsuario_6">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>12</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Núm Personas</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>lb_nombre</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>lb_tipoEmpleado</zorder>
  <zorder>lb_numComandas</zorder>
  <zorder>lb_numPlatillos</zorder>
  <zorder>lb_numPersonas</zorder>
  <zorder>tipoUsuario_4</zorder>
  <zorder>tipoUsuario_5</zorder>
  <zorder>tipoUsuario_6</zorder>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
